### PR TITLE
HTML + basic JavaScript Capstone PR

### PR DESCRIPTION
**Pull request by normainobary2021**
In this Capstone project, I worked on the following:
- Created index.html file for the mobile and desktop version
- Created about.html file for the mobile and desktop version
- Created style.css file for all styling
- Created main.js file to dynamically generate featured speakers section and other custom functionality
- Added all necessary images required in the project.

- The Link to my Recorded Loom Video:
https://www.loom.com/share/a5fd639970ec41b2b8351b64aefb5e37

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  {
+    name: 'Yochai Benkler',
+    photo: './images/p1.jpg',
+    role: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas voluptates ipsum corporis nam magnam, quia nisi molestiae temporibus reiciendis officiis?',
+  },
+  {
+    name: 'Kilnam Chon',
+    photo: './images/p2.jpg',
+    role: '',
```
Comment (vechicin) [inline js/main.js]: This speaker is missing their role, please add it.

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,691 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
```
Comment (vechicin) [inline styles/style.css]: Please change this font to `Lato`, as it is a project requirement.

Screen Shot 2022-08-04 at 12 49 19

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @normainobary2021 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Fantastic project! The code is very organized, the UI/UX is outstanding, and your README is on point. This is, by far, one of the best `The Portal` capstone projects I've seen both as a student and as a reviewer, genuinely fantastic job. Congratulations! 💯 

## Required Changes ♻️

- [ ] I couldn't find the link to the video presentation anywhere. Please add it either to your PR or your README file.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you so much @vechicin for your kind review and awesome feedback. I am really glad that my effort on this project paid off. I am happy about your feedback.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @normainobary2021 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- descriptive video✔️ 
- project design well followed✔️ 
- Validations correctly implemented✔️ 
- project works as expected✔️ 
- No linter errors ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoinn

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you very much @tufoinnkuo10. I am happy about this project

**Pull request merged**